Cython/Build/IpythonMagic.py

```diff
@@ -179,8 +179,8 @@ def f(x):
 
     @magic_arguments.magic_arguments()
     @magic_arguments.argument(
-        '-a', '--annotate', action='store_true', default=False,
-        help="Produce a colorized HTML version of the source."
+        '-a', '--annotate', nargs='?', const=1, type=int,
+        help="Produce a colorized HTML version of the source. Use --annotate=2 to include complete generated C/C++-code."
```

Comment:
It would be nice to at least restrict the option values to `1` or `2` and nothing else, but actually, this option still doesn't feel right to me. When I see `--annotate=2`, I wouldn't know what that does.